resource-managers/kubernetes/lightbend-build/project/plugins.sbt

```diff
@@ -0,0 +1,3 @@
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.0.1")
```

Comment:
I suggest removing the Eclipse plugin. Users of Eclipse can add this to their local `~/.sbt/1.0/...` environments.

Reply:
Sure ;)